buildVars.py

```diff
@@ -59,3 +59,8 @@ def _(arg):
 # Files that will be ignored when building the nvda-addon file
 # Paths are relative to the addon directory, not to the root directory of your addon sources.
 excludedFiles = []
+
+# Base language for the NVDA add-on
+# If the source code, add-on interface and readme file located in the root folder are written in a language other than english, modify this variable as appropriate.
```

Comment:
Please split this line up. While we don't have strict requirement about line lengths I believe 80-90 chars is a sane maximum.